CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,51 @@
+# Contributor Code of Conduct
+
+Our [company values](https://auth0.com/careers/culture) guide us in our day-to-day interactions and decision-making. Our open source projects are no exception. Trust, respect, collaboration and transparency are core values we believe should live and breathe within our projects. Our community welcomes participants from around the world with different experiences, unique perspectives, and great ideas to share.
```

Comment:
We do not need a CODE_OF_CONDUCT. 
And if we really need one, I would prefer to not blindly copy paste it from another company ;)